ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/tests/Tests.scala

```diff
@@ -54,15 +55,15 @@ object Tests {
       new ValueLimitsIT,
       new WitnessesIT,
       new WronglyTypedContractIdIT,
-      new DeeplyNestedValueIT,
     ) ++ (if (supportsExceptions) Vector(new ExceptionsIT, new ExceptionRaceConditionIT)
           else Vector.empty)
 
   val optional: Vector[LedgerTestSuite] =
     Vector(
-      new ParticipantPruningIT,
-      new MultiPartySubmissionIT,
       new CommandDeduplicationOffsetIT,
+      new ContractIdIT,
```

Comment:
Why are these new tests optional?

Reply:
> // Check the Ledger API accepts or rejects non-suffixed contract ID.
> // - Central committer ledger implementations (sandboxes, KV...) may accept non-suffixed CID
> // - Distributed ledger implementations (e.g. Canton) must reject non-suffixed CID

In other words, 
* for Sandbox and KV we should use: 
  "--include=ContractIdIT:Accept"
* for Canton we should use 
  "--include=ContractIdIT:RejectNonSuffixedCid,ContractIdIT:AcceptSuffixedCid"

Reply:
OK, cool. This is good ammunition for allowing ledger-specific tests.